docs/reference-guides/data/data-core-block-editor.md

```diff
@@ -48,6 +48,62 @@ _Returns_
 
 -   `boolean`: Whether the given block type is allowed to be inserted.
 
+### canMoveBlock
+
+Determines if the given block is allowed to be moved.
+
+_Parameters_
+
+-   _state_ `Object`: Editor state.
+-   _clientId_ `string`: The block client Id.
+-   _rootClientId_ `?string`: Optional root client ID of block list.
+
+_Returns_
+
+-   `boolean`: Whether the given block is allowed to be moved.
+
+### canMoveBlocks
+
+Determines if the given blocks are allowed to be moved.
+
+_Parameters_
+
+-   _state_ `Object`: Editor state.
+-   _clientIds_ `string`: The block client IDs to be moved.
+-   _rootClientId_ `?string`: Optional root client ID of block list.
+
+_Returns_
+
+-   `boolean`: Whether the given blocks are allowed to be moved.
+
+### canRemoveBlock
```

Comment:
The canRemoveBlock block should also be used on the transforms verification. If a block can not be removed I should also not be able to transform it into another block and right now we can.

Reply:
This is now resolved with 15fac68